src/components/MajorFilter.jsx

```diff
@@ -1,49 +1,25 @@
-import React, { useState } from 'react';
-import { Autocomplete, Chip, OutlinedInput, IconButton } from '@mui/material';
+import React from 'react';
+import { Chip } from '@mui/material';
 import { MAJORS } from '../types/options';
-import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
 import PropTypes from 'prop-types';
-import ArrowDropUpIcon from '@mui/icons-material/ArrowDropUp';
-import SearchIcon from '@mui/icons-material/Search';
 import useQueryParams from '../lib/useQueryParams';
+import AutocompleteFilter from './AutocompleteFilter';
+import { useTranslation } from 'react-i18next';
 
 function MajorFilter({ majors, onChange }) {
```

Comment:
I thought your plan was to completely get rid of this component and replace its usage with something like:

```jsx
<AutocompleteFilter
  freeSolo
  value={majors}
  onChange={(e, val) => onChange(val)}
  options={[...MAJORS]}
  limit={10}
  placeholder={`${t('enterMajorFilter')}...`}
/>
```

And do something similar for the State filter.